src/component/feature/FeatureToggleListNew/FeatureToggleListNew.styles.ts

```diff
@@ -3,6 +3,9 @@ import { makeStyles } from '@material-ui/core/styles';
 export const useStyles = makeStyles(theme => ({
     tableRow: {
         cursor: 'pointer',
+        '&:hover': {
+            backgroundColor: '#f5f5f5',
```

Comment:
Can we use the color from the theme here, instead of hard-coding.

Reply:
Yess. didn't pay attention to that sorry. I'll update it

Reply:
You can use theme.palette.grey[100-1000]. So for example theme.palette.grey[500] - these are builtin shades of grey from material ui.

Reply:
@FredrikOseberg I changed that in the new commit ✅